train.py

```diff
@@ -157,7 +178,21 @@ def train(config):
     config.argv_update(left_argv)
 
     config.exp_name = basename(args.config).split(".")[0]
-    config.exp_version = datetime.datetime.now().strftime("%Y%m%d_%H%M%S") if not args.exp_version else args.exp_version
+    config.exp_version = (
+        datetime.datetime.now().strftime("%Y%m%d_%H%M%S")
+        if not args.exp_version
+        else args.exp_version
+    )
+    if config.get("wandb", False):
+        run = wandb.init()
```

Comment:
Move this line `if config.get("wandb", False)` just before creation of `WandbLogger` (we will have one less `if config.get("wandb", False)`).